egs2/TEMPLATE/s2t1/README.md

```diff
@@ -0,0 +1,151 @@
+
+More details can be found in our [OWSM](https://arxiv.org/abs/2309.13876) paper (ASRU 2023).
+
+
+## Table of Contents
+
+* [ESPnet2 S2T1 Recipe TEMPLATE](#espnet2-s2t1-recipe-template)
+  * [Table of Contents](#table-of-contents)
+  * [Recipe flow](#recipe-flow)
+    * [1\. Data preparation](#1-data-preparation)
+    * [2\. Speed perturbation](#2-speed-perturbation)
+    * [3\. Wav format](#3-wav-format)
+    * [4\. Remove long or short data](#4-remove-long-or-short-data)
+    * [5\. Generate token list](#5-generate-token-list)
+    * [6\. LM statistics collection](#6-lm-statistics-collection)
+    * [7\. LM training](#7-lm-training)
+    * [8\. LM perplexity](#8-lm-perplexity)
+    * [9\. Ngram-LM training](#9-ngram-lm-training)
+    * [10\. S2T statistics collection](#10-s2t-statistics-collection)
+    * [11\. S2T training](#11-s2t-training)
+    * [12\. S2T inference](#12-s2t-inference)
+    * [13\. S2T scoring](#13-s2t-scoring)
+    * [14\-16\. (Optional) Pack results for upload](#14-16-optional-pack-results-for-upload)
+  * [How to run](#how-to-run)
+    * [OWSM Training](#owsm-training)
+  * [Related works](#related-works)
+
+## Recipe flow
+
+S2T1 recipe consists of 16 stages.
+
+### 1. Data preparation
+
+Data preparation stage.
+
+#### ESPnet format:
+
+It calls `local/data.sh` to creates [Kaldi-style data](https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE#about-kaldi-style-data-directory) directories in `data/` for training and validation sets.
+
+The training data has the following format:
+```
+<sop> prev<sos><category><task><starttime1> utt1<endtime1><starttime2> utt2<endtime2><eos>
+```
+where `<sop>` is a special token denoting the start of prev/prompt sentence. The timestamps are also treated as special tokens because the audio has a fixed length (30s) and resolution (20ms or 40ms). An example looks like:
+
+```
+<sop> I'm going to talk today about energy and climate.<sos><en><transcribe><0.00> And that might seem a bit surprising, because my full-time work at the foundation is mostly about vaccines and seeds, about the things that we need to invent and deliver to help the poorest two billion live better lives.<14.12><15.36> But energy and climate are extremely important to these people; in fact, more important than to anyone else on the planet.<24.26><eos>
+```
+
+During data preparation, three text files are generated:
+- `text` contains the normal target sentence, i.e., the text between `<sos>` and `<eos>`.
+- `text.prev` contains the previous sentence, i.e., the text between `<sop>` and `<sos>`. This might be unavailable at the beginning of a talk. In such cases, a special token `<na>` will be used.
+- `text.ctc` contains the ASR transcript without any special token, which is used for the CTC loss. For ASR utterances, this can be derived from `text`, but for ST utterances, this is in a different language. If the ASR transcription is not available, `<na>` will be used.
+
+
+### 2. Speed perturbation
```

Comment:
Do we need sp?
Maybe we can remove it even from the template in the future.

Reply:
yeah, we may not need it in the future. We didn't use it previously.